### PR TITLE
[EOSF-601] Don't call GA event when defaults get loaded, add lodash to preprints

### PR DESCRIPTION
**Pull request by hmoco**
## Ticket

https://openscience.atlassian.net/browse/EOSF-601

## Purpose
When loading defaults for new preprints, a save action is fired to save the defaults to the preprint in case the user doesn't make any changes (since the defaults are not enforced at the model level). However, since that is not a user action, it should not send a Google analytics request. Also adds lodash so the widget doesn't throw errors. 

## Changes
Add lodash to packages in ember-preprints

### CODE REVIEW

**Comment by hmoco** [comment]
Failing test is a problem from newest develop, I believe. This is fixed in my other PR: https://github.com/CenterForOpenScience/ember-preprints/pull/293

**Comment by jamescdavis** [comment]
PR #293 is now merged.

**Comment by brianjgeiger** [comment]
Build restarted

File: app/controllers/submit.js [hunk 1]
```diff
@@ -400,12 +400,15 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                 licenseValid
             });
 
-            Ember.get(this, 'metrics')
-                .trackEvent({
-                    category: 'dropdown',
-                    action: 'select',
-                    label: `Preprints - ${this.get('editMode') ? 'Edit' : 'Submit'} - Edit License`
-                });
+            if (this.get('initialLicenseChangeMade') || this.get('editMode')) {
```
Comment (pattisdr) [inline app/controllers/submit.js]: Looks good!

**Comment by brianjgeiger** [comment]
Merge conflicts

**Pull request merged**